worlds/yugioh06/__init__.py

```diff
@@ -0,0 +1,538 @@
+import os
+from typing import ClassVar, Dict, Any
+
+import bsdiff4
+import math
+import settings
+
+import Utils
+from BaseClasses import Item, Location, Region, Entrance, MultiWorld, ItemClassification, Tutorial
+from .utils import openFile
+from ..AutoWorld import World, WebWorld
+from .Items import item_to_index, tier_1_opponents, booster_packs, excluded_items, Banlist_Items, core_booster, \
+    challenges, useful, draft_boosters, draft_opponents
+from .Locations import Bonuses, Limited_Duels, Theme_Duels, Campaign_Opponents, Required_Cards, \
+    get_beat_challenge_events, special, collection_events
+from .Opponents import get_opponents, get_opponent_locations, challenge_opponents
+from .Options import Yugioh06Options
+from .Rom import YGO06DeltaPatch, get_base_rom_path, MD5Europe, MD5America
+from .Rules import set_rules
+from .logic import YuGiOh06Logic
+from .BoosterPacks import booster_contents, get_booster_locations
+from .StructureDeck import get_deck_content_locations
+from .RomValues import structure_deck_selection, banlist_ids, function_addresses
+from .Client_bh import YuGiOh2006Client
+
+
+class Yugioh06Web(WebWorld):
+    theme = "stone"
+    setup = Tutorial(
+        "Multiworld Setup Tutorial",
+        "A guide to setting up Yu-Gi-Oh! - Ultimate Masters Edition - World Championship Tournament 2006"
+        "for Archipelago on your computer.",
```

Comment:
This line split renders without a space on the webhost (copied from my local webhost on this branch) `Tournament 2006for Archipelago on`